doc/TRexDataAnalysis.py

```diff
@@ -108,7 +108,8 @@ def analyze_latest_test_results(self):
             test_dates.append(test.latest_result_date)
             test_latest_results.append(test.latest_result)
         self.latest_test_results = pd.DataFrame(
-            {'Date': test_dates, 'Test Name': test_names, 'MPPS\Core (Norm)': test_latest_results},
+            {'Date': test_dates, 'Test Name': test_names,
+             'MPPS/Core (Norm)': test_latest_results},
```

Comment:
I changed this to a forward slash because other locations in that file use a forward slash too (see eg. l150).